percona-server.56/Dockerfile

```diff
@@ -1,35 +1,41 @@
 FROM centos:7
-MAINTAINER Percona Development <info@percona.com>
+
+LABEL org.opencontainers.image.authors="info@percona.com"
 
 RUN groupadd -g 1001 mysql
 RUN useradd -u 1001 -r -g 1001 -s /sbin/nologin \
 		-c "Default Application User" mysql
 
 # check repository package signature in secure way
-RUN export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
-	&& gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
-	&& rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 \
-	&& curl -L -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
-	&& rpmkeys --checksig /tmp/percona-release.rpm \
-	&& yum install -y /tmp/percona-release.rpm \
-	&& rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
-	&& rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY \
-	&& percona-release disable all \
-	&& percona-release enable original release
+RUN set -ex; \
+    export GNUPGHOME="$(mktemp -d)"; \
+    gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A; \
+    gpg --batch --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona; \
+    rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7; \
+    \
+    curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm; \
+    rpmkeys --checksig /tmp/percona-release.rpm; \
+    yum install -y /tmp/percona-release.rpm; \
+    \
+    rm -rf "$GNUPGHOME" /tmp/percona-release.rpm; \
+    rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY; \
+    \
+    percona-release disable all; \
+    percona-release enable original release
 
 # install exact version of PS for repeatability
 ENV PERCONA_VERSION 5.6.47-rel87.0.1.el7
 
-RUN yum install -y \
-		Percona-Server-server-56-${PERCONA_VERSION} \
-		Percona-Server-tokudb-56-${PERCONA_VERSION} \
-		Percona-Server-rocksdb-56-${PERCONA_VERSION} \
```

Comment:
is it intentional?

Reply:
We do not have Percona-Server-rocksdb-56 rpm.